Tests/SentryTests/RedactRegionTests.swift

```diff
@@ -0,0 +1,102 @@
+import Foundation
+import Nimble
+@testable import Sentry
+import XCTest
+#if os(iOS) || os(tvOS)
+class RedactRegionTests: XCTestCase {
```

Comment:
`m`: Those are neat test cases. I think it would make sense to add some test cases when the CGRects don't overlap exactly. For example, what happens if the region to subtract is bigger than the sut region.
For example
```
func testSplitBySubtracting_BottomIsWider() {
    let sut = RedactRegion(rect: CGRect(x: 0, y: 0, width: 100, height: 100), color: .red)
    
    let result = sut.splitBySubtracting(region: CGRect(x: 0, y: 50, width: 150, height: 50))
    
    expect(result.count) == 1
    expect(result.first?.rect) == CGRect(x: 0, y: 0, width: 100, height: 50)
    expect(result.first?.color) == .red
}
```